### PR TITLE
Creator badges

### PR DESCRIPTION
**Pull request by rogeruiz**

### CODE REVIEW

File: api/models/User.js [hunk 1]
```diff
@@ -57,6 +57,12 @@ module.exports = {
       defaultsTo: 0
     },
 
+    // Store the number of assignedTasks
+    //assignedTasks: {
+      //type: 'INTEGER',
+      //defaultsTo: 0,
+    //},
+
```
Comment (ultrasaurus) [inline api/models/User.js]: should we remove this commented out code before checking in?

File: api/models/User.js [hunk 2]
```diff
@@ -68,26 +74,38 @@ module.exports = {
...
+      user.completedTasks = 1 + user.completedTasks;
+
+      this.save( function ( err, u ) {
+
+        if ( err ) { return sails.log.error( err ); }
+
+        Badge.awardForTaskCompletion( task, user, opts );
+
+      } );
+
```
Comment (ultrasaurus) [inline api/models/User.js]: a little nit picky, but `+= 1;` is more readable than `user.completedTasks = 1 + user.completedTasks;`
I think these two code blocks are actually identical

File: api/models/User.js (hunk 1 above)
Comment (rogeruiz) [inline api/models/User.js]: Ah yeah, I'll update it.

File: api/models/User.js (hunk 2 above)
Comment (rogeruiz) [inline api/models/User.js]: Sounds good, I'll revert this as well.

**Comment by ultrasaurus** [comment]
did @jeremiak have the chance to review this?

**Comment by ultrasaurus** [comment]
by the way, I did review all the code, and *think* I understand it and it all looks good to me!  So, we don't need to wait for @jeremiak but would be nice if he's avail

**Pull request merged**